--- a/tests/eds/meson.build.orig	2020-02-04 08:38:51.000000000 +0100
+++ b/tests/eds/meson.build	2020-02-04 09:21:26.009817830 +0100
@@ -102,5 +102,6 @@
   benchmark(_benchmark, benchmark_executable,
     suite: 'EDSBackend',
     env: common_test_environment,
+    timeout: 120,
   )
 endforeach
--- a/tests/lib/gtestdbus.c
+++ b/tests/lib/gtestdbus.c
@@ -104,7 +104,7 @@
   g_idle_add (dispose_on_idle, object);
 
   /* Make sure we don't block forever */
-  timeout_id = g_timeout_add (30 * 1000, on_weak_notify_timeout, &data);
+  timeout_id = g_timeout_add (300 * 1000, on_weak_notify_timeout, &data);
 
   g_main_loop_run (data.loop);
 
